extractor/image/daemon/image_test.go

```diff
@@ -0,0 +1,251 @@
+package daemon
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/docker/docker/api/types"
+	"github.com/google/go-containerregistry/pkg/name"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+
+	"github.com/aquasecurity/testdocker/engine"
+)
+
+func TestMain(m *testing.M) {
+	imagePaths := map[string]string{
+		"index.docker.io/library/alpine:3.10": "../../testdata/alpine-310.tar.gz",
+		"index.docker.io/library/alpine:3.11": "../../testdata/alpine-311.tar.gz",
+		"gcr.io/distroless/base:latest":       "../../testdata/distroless.tar.gz",
+	}
+	opt := engine.Option{
+		APIVersion: "1.38",
+		ImagePaths: imagePaths,
+	}
+	te := engine.NewDockerEngine(opt)
+	defer te.Close()
+
+	os.Setenv("DOCKER_HOST", fmt.Sprintf("tcp://%s", te.Listener.Addr().String()))
+
+	code := m.Run()
+	os.Exit(code)
```

Comment:
```suggestion
	os.Exit(m.Run())
```

Reply:
I've fixed it.
https://github.com/aquasecurity/fanal/pull/96/commits/3558e961a29dd1ab5fae0a14de2c73c89e175a85